src/UpVote.jsx

```diff
@@ -0,0 +1,29 @@
+import { patchVote } from "./api";
+import { useState } from "react";
+
+const UpVote = ({ votes, article_id }) => {
+
+    const [voteChange, setVoteChange] = useState(0);
+
+    const increaseVote = (voteChange) => {
+        setVoteChange((currVote) => currVote + voteChange)
+        patchVote(article_id, voteChange).catch(() => {
+            setVoteChange((currVote) =>
+                currVote - voteChange)
+            alert("Vote failed to submit")
+        })
+    }
+
+    return (
+        <section className="vote">
+            <p>Votes for this article: <span>{votes + voteChange} </span></p>
+            
+            <button onClick={() => increaseVote(1)}>Vote👍🏽</button>
```

Comment:
Emoji's come with some a11y concerns in that they will be read by their literal text. So a screen reader will read this button as `"vote thumbs up"` which doesn't make a lot of sense. 

To fix you can add and `aria-label` which a screen reader will read instead. e.g.
`Vote<span aria-label='up'>👍🏽</span>`